notebook/tests/selenium/quick_selenium.py

```diff
@@ -0,0 +1,22 @@
+from selenium.webdriver import Firefox
+from notebook.notebookapp import list_running_servers
+
+
+def quick_driver():
+    """Quickly create a selenium driver pointing at an active noteboook server.
+
+    Usage example
+    
+    from inside the selenium test directory:
+    
+        import quick_selenium, test_markdown, utils
+        nb = utils.Notebook(test_markdown.notebook(quick_selenium.quick_driver()))
+    """
+    try:
+        server = list(list_running_servers())[0]
+    except IndexError as e:
+        e.message = 'You need a server running before you can run this command'
+    driver = Firefox()
+    auth_url = f'{server["url"]}?token={server["token"]}'
```

Comment:
f strings are great, but we're still supporting Python 3.4 and 3.5 for now, so we'll need to stick with the explicit `.format()` method for now.

Reply:
✓